audformat/core/common.py

```diff
@@ -227,13 +227,22 @@ class DefineBase:
 
     @classmethod
     def assert_has_value(cls, value):
-        attributes = inspect.getmembers(
-            cls, lambda x: not inspect.isroutine(x))
-        valid_values = [a[1] for a in attributes if
-                        not(a[0].startswith('__') and a[0].endswith('__'))]
+        valid_values = cls.values()
         if value not in valid_values:
             raise BadValueError(value, valid_values)
 
+    @classmethod
+    def values(cls):
```

Comment:
Maybe we should choose a more descriptive name like `class_attributes`?

Reply:
But would `class_attributes` not refer to the name of the attribute, or would it refer to its content?

Reply:
Good point. Maybe `class_attribute_values`  then?

Reply:
I renamed it to `attribute_values`